pkg-manager/resolve-dependencies/src/resolveDependencies.ts

```diff
@@ -824,11 +824,10 @@ async function resolveDependenciesOfDependency (
     prefix: options.prefix,
     proceed: extendedWantedDep.proceed || updateShouldContinue || ctx.updatedSet.size > 0,
     publishedBy: options.publishedBy,
-    update,
+    update: update ? options.updateToLatest ? 'latest' : 'in-range' : false,
```

Comment:
"compatible" is the term that we currently use
```suggestion
    update: update ? options.updateToLatest ? 'latest' : 'compatible' : false,
```